src/hyperlight_host/src/hypervisor/hyperv_linux.rs

```diff
@@ -271,15 +271,10 @@ mod debug {
 /// and functional.
 #[instrument(skip_all, parent = Span::current(), level = "Trace")]
 pub(crate) fn is_hypervisor_present() -> bool {
-    match Mshv::open_with_cloexec(true) {
-        Ok(fd) => {
-            unsafe {
-                libc::close(fd);
-            } // must explicitly close fd to avoid a leak
-            true
-        }
-        Err(e) => {
-            log::info!("Error creating MSHV object: {:?}", e);
+    match Mshv::new() {
+        Ok(_) => true,
```

Comment:
Why don't you need to close the fd anymore? I added this a while back because we were getting an "out of resources" error due to leaking this fd in long running intensive tests.